src/Compilers/CSharp/Portable/Symbols/PublicModel/DiscardSymbol.cs

```diff
@@ -34,6 +37,7 @@ ITypeSymbol IDiscardSymbol.Type
         CodeAnalysis.NullableAnnotation IDiscardSymbol.NullableAnnotation => _underlying.TypeWithAnnotations.ToPublicAnnotation();
 
         protected override void Accept(SymbolVisitor visitor) => visitor.VisitDiscard(this);
+        [return: MaybeNull]
```

Comment:
This doesn't seem appropriate? The return will be based off of the visitor result supplied?

Reply:
If only `DefaultVisit` was abstract...

https://github.com/dotnet/roslyn/blob/c6d423199dc9ec962eafa8a993436854dc209f9a/src/Compilers/Core/Portable/Symbols/SymbolVisitor%601.cs#L21-L27